config/software/ruby.rb

```diff
@@ -228,8 +238,9 @@
   env["PKG_CONFIG"] = "/bin/true" if aix?
 
   configure(*configure_command, env: env)
-  make "-j #{workers}", env: env
-  make "-j #{workers} install", env: env
+  pmake = "-j #{workers}"
+  make pmake, env: env
+  make "#{pmake} install", env: env
 
   if windows?
```

Comment:
This will break when we build against the toolchain since it makes an incorrect assumption about where msys is installed. See my related questions in https://github.com/chef-cookbooks/omnibus/pull/191